keel-igor/src/main/kotlin/com/netflix/spinnaker/keel/igor/BuildLifecycleMonitor.kt

```diff
@@ -33,7 +37,7 @@ class BuildLifecycleMonitor(
   override val lifecycleConfig: LifecycleConfig,
   val objectMapper: ObjectMapper,
   val artifactMetadataService: ArtifactMetadataService,
-  val front50Service: Front50Service,
+  val front50Service: Front50Service?,
```

Comment:
Why is `front50Service` optional?

Reply:
Because it's not actually a required dependency of keel, and I don't think this is important to make it so. If we can't look up app details in front50 here, because it's not wired in, then we will fallback to the jenkins URL. that seemed fine to me.